plugins/module_utils/ism.py

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8 -*-
+# Copyright (c), Inspur isib-group, 2020
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+try:
+    from ansible_collections.community.general.plugins.module_utils.inspur_sdk import ism
+    ism_temp = True
+except ImportError:
+    ism_temp = False
+from ansible.module_utils.basic import env_fallback
+from ansible.module_utils.six import iteritems
+
+
+ism_provider_spec = {
+    'host': dict(type='str'),
+    'username': dict(type='str', fallback=(env_fallback, ['ANSIBLE_NET_USERNAME'])),
+    'password': dict(type='str', fallback=(env_fallback, ['ANSIBLE_NET_PASSWORD']), no_log=True),
+}
+ism_argument_spec = {
+    'provider': dict(type='dict', options=ism_provider_spec),
+}
+ism_top_spec = {
+    'host': dict(type='str'),
+    'username': dict(type='str'),
+    'password': dict(type='str', no_log=True),
+}
+ism_argument_spec.update(ism_top_spec)
```

Comment:
Here you add `host`, `username` and `password` also to the top level. And for these you do not have entries in the docs fragment.